packages/xo-server-backup-reports/src/index.js

```diff
@@ -243,19 +243,21 @@ class BackupReportsXoPlugin {
           const operationInfoText = []
           if (operationLog.status === 'success') {
             const size = operationLog.result.size
-            if (operationLog.message === 'merge') {
-              globalMergeSize += size
-            } else {
-              globalTransferSize += size
+            if (size > 0) {
```

Comment:
The main issue here is the duration, not the size.

Maybe the correct fix would be to handle it in `formatSpeed`.